azurerm/internal/services/managedservices/tests/data_source_registration_definition_test.go

```diff
@@ -0,0 +1,72 @@
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccDataSourceAzureRMRegistrationDefinition_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_registration_definition", "test")
+	// Multiple tenants are needed to test this resource. 
+	// Second tenant ID needs to be set as a enviornment variable ARM_SECOND_TENANT_ID.
+	secondTenantID := os.Getenv("ARM_SECOND_TENANT_ID")
```

Comment:
The resources in this PR needs multiple tenant to run the scenario. I ran these tests after setting the environment variable ARM_SECOND_TENANT_ID. Is there a way to setup the second tenant for these tests?